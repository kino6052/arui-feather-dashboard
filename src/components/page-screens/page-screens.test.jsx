import React from 'react';
import { shallow, mount } from 'enzyme';
import ScreenPage from './page-screens';

describe('Screen page', () => {
    /* https://github.com/airbnb/enzyme/blob/master/docs/api/shallow.md */
    it('should render shallowly', () => {
        expect(shallow(<ScreenPage pageKey='0' />)).toMatchSnapshot();
    });
    /* https://github.com/airbnb/enzyme/blob/master/docs/api/render.md */
    it('should render deeply', () => {
        expect(mount(<ScreenPage pageKey='1' />)).toMatchSnapshot();
    });
});
