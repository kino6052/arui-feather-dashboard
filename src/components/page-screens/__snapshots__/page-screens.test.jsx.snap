// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen page should render deeply 1`] = `
<ScreenPage
  match={Object {}}
  pageKey="1"
  routeParams={null}
>
  <div>
    <Heading
      size="xl"
    >
      <h1
        className="heading heading_size_xl heading_theme_alfa-on-white"
      >
        Экран 2
      </h1>
    </Heading>
    <Paragraph>
      <p
        className="paragraph paragraph_theme_alfa-on-white"
      >
        В 2015 году улучшился показатель стоимости риска Банковской
                группы «Альфа-Банк» — уменьшение с 3.96% до 3.12%, благодаря
                снижению темпов роста проблемной задолженности. При этом
                традиционно консервативный подход в анализе рисков отразился
                в увеличении ставки резервирования с 5,8% до 6,5%.
      </p>
    </Paragraph>
  </div>
</ScreenPage>
`;

exports[`Screen page should render shallowly 1`] = `
<div>
  <Heading
    size="xl"
  >
    Экран 1
  </Heading>
  <Paragraph>
    Чистая прибыль Банковской группы выросла в 15 раз по сравнению с предыдущим годом и
                достигла 480 млн. долларов США (против 33 млн. долларов США по итогам 2014 года).
                Операционная прибыль до создания резервов составила 2 257 млн. долларов США
                (снижение на 12,9% по сравнению с 2014 годом объясняется ослаблением среднего
                курса рубля — на 58,7% за год). В рублевом эквиваленте операционная прибыль
                до создания резервов выросла в 2015 году более чем на треть (+38,2%).
  </Paragraph>
</div>
`;
